libnavigation-base/src/main/res/values-fr/strings.xml

```diff
@@ -3,6 +3,7 @@
     <!-- Time formatting -->
     <plurals name="mapbox_number_of_days" tools:ignore="MissingTranslation">
         <item quantity="one" tools:ignore="ImpliedQuantity">jour</item>
+        <item quantity="many">jours</item>
```

Comment:
What was the Linter issue here? Asking becuase this may impact Transifex translations.

Reply:
Yeah, lint was failing because of this. For some reason only for fr, although for some other languages "many" is not present as well.
How may that impact translations? It's the same as "other" and before my change "other" option was used for "many", right?

Reply:
Alternatively, I can add "MissingQuantity" to `tools:ignore` if that's an issue, but is it?

Reply:
We should sync it with Transifex translations, I see that last time we opted for `tools:ignore` for `"one"` because having problems with Transifex and the base `strings.xml` (not getting the plurals correctly).

Reply:
I've pulled in the latest translations from Transifex in https://github.com/mapbox/mapbox-navigation-android/pull/6265

Could you rebase here after https://github.com/mapbox/mapbox-navigation-android/pull/6265 lands?

Reply:
Done, thanks.